dev/NavigationView/NavigationView.cpp

```diff
@@ -585,7 +585,8 @@ void NavigationView::UpdateAdaptiveLayout(double width, bool forceSetDisplayMode
         MUX_FAIL_FAST();
     }
     if (!forceSetDisplayMode && m_InitialNonForcedModeUpdate) {
```

Comment:
nit: add a blank line between this if statement and the else block above it so that the two blocks are visually separated.